PrinceXML.Wrapper/Prince.cs

```diff
@@ -505,7 +505,8 @@ private List<string> GetJobCommandLine(string logType)
 
             foreach ((string k, string v) in Options)
             {
-                cmdLine.Add(ToCommand(k, v));
+                string val = v == "" ? null : v;
```

Comment:
Is this going to cause trouble for any options where we _want_ to allow empty string? `--user-password` perhaps?

Reply:
Right, hadn't considered that. Looks like it's back to the original idea of updating the documentation then.